.travis.yml

```diff
@@ -43,6 +49,12 @@ matrix:
   - rvm: default
     sudo: required
     services: docker
+    env: BEAKER_set="debian-8-x86_64-docker"
+    bundler_args:
+    script: bundle exec rspec spec/acceptance/*_spec.rb
+  - rvm: default
+    sudo: required
+    services: docker
```

Comment:
after modulesync, this will either have to be put into our modulesync_config/ or in .sync.yml